src/FilterList.js

```diff
@@ -1,17 +1,40 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {ResultPropTypes} from './ResultPropTypes'
+import _ from 'lodash'
 
-const FilterList = ({filteredResults, resultsMessage, ResultsHeaderClass, ResultElementClass}) => {
-  const filteredElements = filteredResults.map((result) => result.element)
+class FilterList extends React.Component {
+  constructor(props) {
+    super(props)
 
-  return (
-    <div>
-      <h4>{resultsMessage}</h4>
-      { filteredElements.length && ResultsHeaderClass && <ResultsHeaderClass /> }
-      { filteredElements.map((element, index) => <div key={index}><ResultElementClass {...element}/></div>) }
-    </div>
-  )
+    this.state = {
+      ascending: false
+    }
+
+    this.sortTable = this.sortTable.bind(this)
```

Comment:
I think `sortTable` is not the best name since the results aren’t necessarily a table. Can you think of a better name?